flaml/ml.py

```diff
@@ -516,31 +532,20 @@ def evaluate_model_CV(
         )
         if weight is not None:
             fit_kwargs["sample_weight"] = weight
-        valid_fold_num += 1
         total_fold_num += 1
-        total_val_loss += val_loss_i
+        val_loss_folds.append(val_loss_i)
         if log_training_metric or not isinstance(eval_metric, str):
-            if isinstance(total_metric, dict):
-                total_metric = {k: total_metric[k] + v for k, v in metric_i.items()}
-            elif total_metric is not None:
-                total_metric += metric_i
-            else:
-                total_metric = metric_i
+            if isinstance(metric_i, dict):
+                log_metric_folds.append(metric_i)

```

Comment:
1. remove the "if" condition since we need to append metric_i to the list anyway.
2. metric_i can be a single float number as well. So the list of `log_metric_folds` can contain dicts or floats or Nones.

Reply:
This comment hasn't been addressed.